lib/v2/screens/settings/components/settings_card.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/providers/services/navigation_service.dart';
+
+/// SETTINGS CARD
+class SettingsCard extends StatelessWidget {
+  /// Card icon
+  final IconData icon;
+
+  /// The text title in the first row
+  final String title;
+
+  /// The text value next to the title
+  final String titleValue;
+
+  /// To set a different color to titleValue
+  final Color titleValueColor;
+
+  /// The descrption text in the second row
+  final String descriptionText;
+
+  /// Use if the second row is a widget
+  final Widget descriptionWidget;
+
+  /// Route to which the card navigates.
+  final String route;
```

Comment:
dont pass the string to navigation here. 
Pass a handler (callBack, onTap) for a tap event, and handle the tap on the caller widget.

Reply:
I did 'nt expose the onTap because the only thing implemented in all cards is navigation and that is why I only require the route to avoid repeating the navigation code.

Reply:
u are right, leave it as is.